### ISSUE TITLE
Logspam of: CalculateTankBoiloff found NaN 

### ISSUE THREAD

**Issue by JebsSY** [KSP-RO/RealFuels#329]
Hello together,

on a fresh CKAN KSP 1.12.5 RP-1 Express-Install (latest v1 and also latest v2) my LEM starts log spamming
"[ERR 11:05:40.007] RF: CalculateTankBoiloff found NaN part.temperature on ROC-LEMAscentBDB (Part)"
after the first time warp and return from time warp when using MLI.

https://github.com/KSP-RO/RealFuels/blob/8da9a8012897b7cb442066704563873b5f73be62/Source/Tanks/ModuleFuelTanksRF.cs#L216C38-L216C68

I was debugging the issue and found a easy and reliable way to reproduce:
- fresh install
- start sandbox game
- select LEM ascent stage OR a "procedural isogrid tank and add water"
- select MLI 1 or more
- attach 2 modular girder segments

After spawning check thermal debug info.
-Cond flux is NaN kW.
-go to physicless timewarp 4 clicks ore more
-stop timewarp
-Now Cond Flux is 0.0kW but now internal Temp is NaN and the log spam starts.

So, basically, 4 conditions need to met for this bug:
- f.e. water must be in the tank
- MLI level must be != 0
- 2 parts need to be attached
- timewarp

I assume internal temp get's NaN because it is somehow calculated from the cond flux, which is NaN after spawn.

So the big question is, why is cond flux NaN after spawn when there's water+MLI+some parts attached?!
That I wasn't able to figure out, yet.

BR
Jebs_SY

![KSP1](https://github.com/KSP-RO/RealFuels/assets/24717622/3ef99499-d8dc-42c1-bc35-f43991f69bbc)
![KSP2](https://github.com/KSP-RO/RealFuels/assets/24717622/900771fd-1d97-4ae8-ad13-71c144bda2a7)

**Comment by Starwaster** [comment]
There used to be code that guards against NaN temperatures, and problems like this tended to sort themselves out or at least be prevented from contaminating part temperatures. Unfortunately, that safeguard code seems to have been excised from the current code base.

**Comment by NathanKell** [comment]
KSP itself should be guarding against NaN conductive fluxes. Does this go away without MLI?

**Comment by JebsSY** [comment]
Hello Nathan,
yes, when setting MLI to 0, the problem does not occur.
My workaround for now is to comment out vsp and conductivity values for water and wastewater in RealFuels\Resources\ResourceHsps.cfg 
BR
Jebs_SY
